armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -567,14 +567,15 @@ class TestSpecifyTargetComponent(unittest.TestCase):
     """verify specifyTargetComponent method is properly updating _componentDeterminesBlockHeight"""
 
     def setUp(self):
-        Base.setUp(self)
+        self.obj = AxialExpansionChanger()
         self.a = buildTestAssemblyWithFakeMaterial(name="FakeMatException")
         self.obj.setAssembly(self.a)
+        # need an empty dictionary because we want to test for the added component only
+        self.obj.expansionData._componentDeterminesBlockHeight = (
+            {}
+        )  # pylint: disable=protected-access
```

Comment:
@john-science is the pylint comment necessary here? @albeanth and I aren't sure

Reply:
I am not particularly bothered with `pylint` concerns in unit tests, so I tend to put one big `pylint:disable` at the top of each file:

https://github.com/terrapower/armi/blob/e0dd171ff4fa1433f59e15420a8dcc7428b909af/armi/reactor/converters/tests/test_geometryConverters.py#L16

Seems easier.